.github/buildomat/jobs/check-features.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#:
+#: name = "check-features (helios)"
+#: variety = "basic"
+#: target = "helios-2.0"
+#: rust_toolchain = true
+#: output_rules = []
+
+# Run cargo check on illumos with feature-specifics like `no-default-features`
+# or `feature-powerset`.
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+cargo --version
+rustc --version
+
+# NOTE: This version should be in sync with the recommended version in
+# ./dev-tools/xtask/src/check-features.rs.
+CARGO_HACK_VERSION='0.6.28'
+
+#
+# Set up our PATH for use with this workspace.
+#
+source ./env.sh
+
+banner prerequisites
+ptime -m bash ./tools/install_builder_prerequisites.sh -y
+
+banner check
+export CARGO_INCREMENTAL=0
+ptime -m cargo check --workspace --bins --tests --no-default-features
+RUSTDOCFLAGS="--document-private-items -D warnings" ptime -m cargo doc --workspace --no-deps --no-default-features
```

Comment:
Is there a point to doing this -- is this not covered by `cargo hack`? If so, please put in a comment explaining why.

Reply:
Good point. We can just use `hack` for it all.